app/scripts/modules/core/src/managed/ArtifactRow.module.css

```diff
@@ -3,8 +3,26 @@
   display: block;
   cursor: pointer;
 }
-.ArtifactRow:not(.selected):hover {
-  background-image: linear-gradient(to bottom, rgba(160, 180, 220, 0.15), rgba(160, 180, 220, 0.15));
+
+.ArtifactRow:not(.selected):hover,
+.ArtifactStatus:not(.selected).hovered {
+  /*TODO: Change the color and remove opacity as it is affecting the border around status bubble*/
+  background-color: rgba(160, 180, 220, 0.15);
```

Comment:
@gcomstock, Could we get a color here without the opacity? The opacity is messing up the border around status bubble in the hover state as you can see below.

![image](https://user-images.githubusercontent.com/357832/85589073-1b22be00-b5f8-11ea-8e43-bbe55c8707eb.png)

Reply:
Yes! #f1f4fa

Reply:
Some feedback on these:

you can reduce the border width to 2px for each bubble.
please add some margin to the right of the +2 bubble, so that bubble takes up the same width as a normal bubble. Maybe a mix of margin and making the gray bubble a bit larger. This way, the bubbles form columns that line up visually:

![Screen Shot 2020-06-24 at 4 53 47 PM](https://user-images.githubusercontent.com/3046463/85638767-a2455580-b63b-11ea-83af-3b728c4b6f6d.png)

@erikmunson , @vigneshm do you think a status bubble for an active deployment is necessary when the environment bar will also be flashing? Im inclined to think less is more, and just the environment bar is enough. But it does break a bit of a pattern we're trying to establish. Let me know.

Thanks for all the work Vignesh! Looks great otherwise.

Reply:
> do you think a status bubble for an active deployment is necessary when the environment bar will also be flashing? Im inclined to think less is more, and just the environment bar is enough. But it does break a bit of a pattern we're trying to establish.

Good point. The flashing bar must be enough then. I can remove the deploying status icon.

I've reduced each status bubble's width, increased the size of the `count` bubble and added some right margin to make them all aligned. Let me know if this looks fine.

![image](https://user-images.githubusercontent.com/357832/85645791-95326180-b64f-11ea-992d-c7aa6ba31238.png)

![image](https://user-images.githubusercontent.com/357832/85645845-b2673000-b64f-11ea-89f6-5958b5ec10e9.png)

Reply:
Looks perfect, thank you!!